Star Trek Adventurers 2d20/Star Trek Adventures 2d20.css

```diff
@@ -0,0 +1,162 @@
+/* === GENERAL === */
+.charsheet {
+    min-width: 820px;
+    background: #FFFFFF;
+    background-image: url("https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Star Trek Adventurers 2d20/Sheet_blank.PNG");
```

Comment:
Might be a better idea to host this image somewhere central like imgur. Trying to test this locally means it needs to change every time.